### ISSUE TITLE
[Feature] Add success screen confirmation when paying webln invoices

### ISSUE THREAD

**Issue by jakubszwedo** [getAlby/lightning-browser-extension#1960]
### Feature description

Currently, when paying Webln called invoices, Alby does not show any in-extension confirmation and only relays on in-browser toast confirmations.

Unfortunately, this means all users who have notifications turned off for their browsers (or are in "focus" mode for example)  never see any visual confirmation of their payment from Alby when paying webln invoices.

This gets worse if websites when users is paying does not show any visual confirmation as well - in that case users have 0 visual clues if the payment went through at all. That requires users to open the extension and check outgoing transactions. leads to bad experience of confusion and clunkiness. Good example of such behaviour is tipping taggers on btcmap.org

### Describe the solution

I propose a quick, 2 or 3 seconds in-extension success confirmation screen. I can see two ways of implementation, depending on technical constrains: 

1. If it’s possible for Alby to detect user notifications settings, then: 
- If user has notifications on, show only browser toast messages 
- If user has notifications turned off, display in-extension success confirmation

2. If we can't detect users settings, then I suggest we show only in-extension confirmation.

The quickest implementation of this is to take current payment confirmation screen that Alby displays when we make payments directly in-extension and just a simple disappearing bar that will indicate that the window will close soon.

You can see how it could look like on [this prototype](https://www.figma.com/proto/AFU2lTs5H7UfEyfc4Nz6hk/Success-screen-for-webln-payments?page-id=0%3A1&node-id=6%3A530&scaling=min-zoom&starting-point-node-id=6%3A315).

### Describe alternatives

Nagging all places that implemented Webln to add visual confirmations when invoice is paid.

### Are you working on this?

None

**Comment by jakubszwedo** [comment]
Here's a design ready for implementation, for both WebLN payments as non-WebLN payments as well:

<img width="872" alt="Screenshot 2023-02-02 at 16 46 12" src="https://user-images.githubusercontent.com/68239231/216372105-a532f6f7-3581-4164-a360-2c7a413e0e0c.png">

You can see animated SVG here of both flows:

1. [WebLN payments flow](https://www.figma.com/proto/AFU2lTs5H7UfEyfc4Nz6hk/Success-screens?page-id=84%3A1320&node-id=84%3A1321&viewport=1267%2C611%2C0.19&scaling=min-zoom&starting-point-node-id=84%3A1321&show-proto-sidebar=1)

When paying WebLN, confirmation screens appear with spinner animation, and when it ends (1800ms), extension window dissapear.

2. [Non-WebLN payments flow
](https://www.figma.com/proto/AFU2lTs5H7UfEyfc4Nz6hk/Success-screens?page-id=84%3A1320&node-id=87%3A1533&viewport=1267%2C611%2C0.19&scaling=min-zoom&starting-point-node-id=87%3A1533&show-proto-sidebar=1)

This screen replaces current success screen. It does not close, until closed manually by the user.

Proposed implementation details:

<img width="923" alt="Screenshot 2023-02-02 at 16 50 31" src="https://user-images.githubusercontent.com/68239231/216373263-80ce4e9f-a102-4d0f-a3f9-803ec7f2d18b.png">

Note: crossed zeroes in Inter is not a must, but a nice have here (it increases numbers legibility) 

[Link to Figma file
](https://www.figma.com/file/AFU2lTs5H7UfEyfc4Nz6hk/Success-screens?node-id=84%3A1320&t=EVorSyefCfkv48yY-0)

**Comment by jankoegel** [comment]
> Note: crossed zeroes in Inter is not a must, but a nice have here (it increases numbers legibility)

They should work with `font-feature-setting: 'zero' on;` in CSS.

Demo from the `website`:
![image](https://user-images.githubusercontent.com/100707419/216380110-b8dd33d9-b6a1-4dba-b221-649931397ab0.png)

**Comment by Ladyperpetua** [comment]
can i please be assigned this issue? @stackingsaunter

**Comment by Ladyperpetua** [comment]
hi @stackingsaunter good afteroon...i've been working on the design of this feature....im quite confused about the disappearing extension window, do you mean you want the toast message that pops up to disappear

**Comment by Ladyperpetua** [comment]
could i show you an example of what i did and you could explain more on what you mean because i'm still quite
 confused about the disappearing part.

**Comment by stackingsaunter** [comment]
Thanks for tackling this @Ladyperpetua! 

When user is transacting in the popup (open extension and transacting from there), there should not be dissapearing or autoclose, and just a close button.

But when transaction is initiated from WebLN (meaning users is on a website like stacker.news or lnmarkets or any other website that has WebLN implemented) and website initiates the payment, the payment screens opens in a window, like int the same like in the prototype. In that case, when user pays, they should see confirmation and it should dissapear (autoclose) soon.

This shouldn't be a toast message for the reason that some users have notifications turned off, which means they will not see a toast message. So the confirmation is in the extension itself. 

Does that make it clear?

**Comment by Ladyperpetua** [comment]
Yes, its much clearer now, thank you....I already implemented that and I will send a PR soon
